### ISSUE TITLE
Review how dates are stored in the iso19115 xml

### ISSUE THREAD

**Issue by fostermh** [cioos-siooc/metadata-xml#68]
I think we may be storing dates in the iso19115 xml incorrectly. 

We currently put all dates under `mdb:MD_Metadata/mdb:dateInfo/cit:CI_Date/`. As I understand it this path is for dates related to the Metadata. For example the creation, publication, and revision of the metadata record. While `mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:citation/cit:CI_Citation/cit:CI_Date/` seems like the place for dates related to the data it’s self.

Thus something like the 'creation' of a dataset or 'start date' of collection should probly be under `mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:citation/cit:CI_Citation/cit:CI_Date/`

from https://wiki.esipfed.org/MD_Metadata item 6 - mdb:dateinfo:
```
Date(s) associated with the metadata record. Highly recommend revisiting the metadata content annually to ensure that all the information is still relevant.
Creation date must be provided, others can also be provided
Use ISO 8601 extended format: YYYY-MM-DD or YYYY-MM-DDTHH:MM:SS.
```

from https://wiki.esipfed.org/CI_Citation_for_MD_DataIdentification item 3 - date: 
```
Publication date identifies when the resource was issued and made publicly available.
Creation date identifies when the resource was created or first date of data collection.
Revision date identifies when the resource was changed.
If the publication or creation date is "unknown" or "inapplicable" then use the gco:nilReason attribute instead of gco:Date. Whenever possible, also include both creation date and revision date.
```

some other links:
https://wiki.esipfed.org/CI_Date
https://wiki.esipfed.org/CI_Citation

**Comment by n-a-t-e** [comment]
Sounds good, so we can put metadata creation date (when the metadata record is created or updated) in `mdb:MD_Metadata/mdb:dateInfo/cit:CI_Date/`

And then put the data collected start date in two places - 
`mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:citation/cit:CI_Citation/cit:CI_Date`
and 
`mdb:MD_Metadata/mdb:identificationInfo/mri:MD_DataIdentification/mri:extent/gex:EX_Extent/gex:temporalElement`

**Comment by n-a-t-e** [comment]
This should be fixed by 65fe836fbd9095cadee24c17ee1c9a8e2b861f4b
